sklearn/datasets/_samples_generator.py

```diff
@@ -163,12 +168,7 @@ def make_classification(
     References
     ----------
     .. [1] I. Guyon, "Design of experiments for the NIPS 2003 variable
-           selection benchmark", 2003.
-
-    See Also
-    --------
-    make_blobs : Simplified variant.
-    make_multilabel_classification : Unrelated generator for multilabel tasks.
+           selection benchmark", 2003.  
```

Comment:
Same here regarding linting issues:

```suggestion
           selection benchmark", 2003.
```

Reply:
Hi, thanks for the review... I corrected the error on _samples_generator.py, but I'm not sure how to proceed with the PR. Do I close it and start all over with a new PR?

Reply:
Answering my own question: I think I have to push to my fork, and this will update the PR. In case this is not correct, please let me know. This is my first PR ever, so thank you for your patience.

Reply:
That is correct. Pushing to the branch on your fork will update the PR.